app/src/main/res/layout/fragment_profile_stats.xml

```diff
@@ -422,6 +422,26 @@
             </LinearLayout>
         </LinearLayout>
 
+
+        <LinearLayout
+            android:id="@+id/not_enrolled"
```

Comment:
we usually follow https://jeroenmols.com/blog/2016/03/07/resourcenaming/ naming convention while naming IDs, layout files. In this case we should use 'll_not_enrolled' . 
Prefix for other layouts that we are following are: 
- Buttons - 'btn_'
- Textviews - 'tv_'
- ImageViews - 'iv_'  
- RecyclerView - 'rv_'
- RelativeLayout - 'rl_'

Reply:
Updated.